e2e/testcases/death/declaration/death-declaration-11.spec.ts

```diff
@@ -0,0 +1,1091 @@
+  expectAddress,
+  expectTextWithChangeLink,
+  formatDateObjectTo_ddMMMMyyyy,
+  getRandomDate,
+  goToSection,
+  joinValuesWith,
+  login
+} from '../../../helpers'
+import faker from '@faker-js/faker'
+
+test.describe.serial('11. Death declaration case - 11', () => {
+  let page: Page
+  const declaration = {
+    deceased: {
+      name: {
+        firstNames: faker.name.firstName('male'),
+        familyName: faker.name.lastName('male')
+      },
+      gender: 'Unknown',
+      age: 45,
+      nationality: 'Farajaland',
+      identifier: {
+        id: faker.random.numeric(10),
+        type: 'Birth Registration Number'
+      },
+      maritalStatus: 'Widowed',
+      address: {
+        country: 'Guam',
+        state: faker.address.state(),
+        district: faker.address.county(),
+        town: faker.address.city(),
+        addressLine1: faker.address.county(),
+        addressLine2: faker.address.streetName(),
+        addressLine3: faker.address.buildingNumber(),
+        postcodeOrZip: faker.address.zipCode()
+      }
+    },
+    event: {
+      manner: 'Suicide',
+      date: getRandomDate(50, 60),
+      cause: {
+        established: true,
+        source: 'Verbal autopsy',
+        description: 'Hanging from ceiling'
+      },
+      place: "Deceased's usual place of residence"
+    },
+    informantType: 'Daughter',
+    informantEmail: faker.internet.email(),
+    informant: {
+      name: {
+        firstNames: faker.name.firstName('female'),
+        familyName: faker.name.lastName('female')
+      },
+      age: 17,
+      birthDate: getRandomDate(5, 200),
+      nationality: 'Malawi',
+      identifier: {
+        id: faker.random.numeric(10),
+        type: 'Passport'
+      },
+      address: {
+        sameAsDeceased: false,
+        country: 'Farajaland',
+        province: 'Chuminga',
+        district: 'Nsali',
+        urbanOrRural: 'Urban',
+        town: faker.address.city(),
+        residentialArea: faker.address.county(),
+        street: faker.address.streetName(),
+        number: faker.address.buildingNumber(),
+        postcodeOrZip: faker.address.zipCode()
+      }
+    },
+    spouse: {
+      name: {
+        firstNames: faker.name.firstName('female'),
+        familyName: faker.name.lastName('female')
+      },
+      age: 42,
+      nationality: 'Farajaland',
+      identifier: {
+        id: faker.random.numeric(10),
+        type: 'Birth Registration Number'
+      },
+      address: {
+        sameAsDeceased: false,
+        country: 'Farajaland',
+        province: 'Chuminga',
+        district: 'Nsali',
+        urbanOrRural: 'Rural',
+        village: faker.address.county()
+      }
+    }
+  }
+  test.beforeAll(async ({ browser }) => {
+    page = await browser.newPage()
+  })
+
+  test.afterAll(async () => {
+    await page.close()
+  })
+
+  test.describe('11.1 Declaratin started by FA', async () => {
+    test.beforeAll(async () => {
+      await login(page, 'k.bwalya', 'test')
```

Comment:
We could move the test user credentials to some shared utils file, what do you think